aait/backend/group-2B/Application/Features/Users/Requests/Commands/UserDeletCommand.cs

```diff
@@ -0,0 +1,14 @@
+
+using Application.DTOs.Users;
+using MediatR;
+using SocialSync.Application.Common.Responses;
+
+namespace Application.Features.Users.Requests.Commands
+{
+    public class UserDeleteCommand : IRequest<CommonResponse<int>>
+    {
+
+        public required UserDeletDto UserdleteDto {get;set;}
```

Comment:
Correct naming is crucial to improve readability. In this case, `UserDeletDto` should be `UserDeleteDto`